app/controllers/app/comments_controller.rb

```diff
@@ -0,0 +1,6 @@
+class App::CommentsController < ApplicationController
```

Comment:
<sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Not covered lines: 1 to 6

Reply:
Tests creados!